src/main/java/com/spotify/confidence/EventSenderEngineImpl.java

```diff
@@ -91,13 +91,25 @@ public void emit(
     LockSupport.unpark(pollingThread);
   }
 
+  @Override
+  public void flush() {
+    sendQueue.add(Event.newBuilder().setEventDefinition("manual_flash").build());
```

Comment:
wdyt about only calling `emit()`?
Relying on all that stuff to handle the unparking etc?

Reply:
I wanted to avoid the `intakeClosed` check, since flush should work even in that case

Reply:
That makes perfect sense!
We could add some logging to this function. But nothing stopping.